src/mmw/js/src/compare/views.js

```diff
@@ -1030,6 +1131,7 @@ function getGwlfeTabs(scenarios) {
             name: 'Water Quality',
             table: qualityTable,
             charts: qualityCharts,
+            selections: qualitySelections,
```

Comment:
The hydrology tab will only show the selection dropdown for the table and not for the line chart. Given that, I think it makes sense to split this into two variables -- `chartSelections` and `tableSelections` -- and redo the check on whether to display it to check on the one for visualization type.

Reply:
This is potentially not blocking, btw -- if we agree to adjust it on the fly we can leave it as is -- or we can make the update quickly here and then adjust the `if (selections)` check above to be `if (chartSelections || tableSelections)` and then determine how to handle each specific case later.

Reply:
I did think about that briefly, although we may have to figure out a way to keep `chartSelections` and `tableSelections` in sync for Water Quality. I'll think more about this.

Reply:
Would it work to have water quality chart and table selections share a single collection rather than making a new one for each?

Reply:
Another way to handle it, btw, is to treat hydrology line charts as a special case and then pass a bool somewhere to suppress the dropdown only for that view.